backend/climateconnect_api/management/commands/create_test_data.py

```diff
@@ -0,0 +1,106 @@
+    print("Creating user availabilities...")
+    for i in range(number_of_rows):
+        name = "{}-{} hours per week".format(i, i+1)
+        key = name.replace(" ", "")
+        if not Availability.objects.filter(name=name).exists():
+            Availability.objects.create(name=name, key=key)
+        else:
+            print("{} user availability already exists.".format(name))
+
+
+def create_roles_test_data():
+    # Creating 2 roles here
+    print("Creating project roles...")
+
+    if not Role.objects.filter(name='Creator').exists():
+        Role.objects.create(
+            name='Creator', role_type=Role.ALL_TYPE
+        )
+        print("Creator role is successfully created.")
+    else:
+        print("Creator role already exists.")
+    
+    if not Role.objects.filter(name='Member').exists():
+        Role.objects.create(
+            name='Member', role_type=Role.READ_ONLY_TYPE
+        )
+        print("Member role successfully created.")
+    else:
+        print("Member role already exists.")
+
+
+def create_project_status_test_data():
+    # Creating 2 project status
+    print("Creating project status...")
+    if not ProjectStatus.objects.filter(name='In Progress').exists():
+        ProjectStatus.objects.create(name='In Progress', has_end_date=True, has_start_date=True)
+        print("In Progress status created.")
+    else:
+        print("In Progress project status already exists.")
+    
+    if not ProjectStatus.objects.filter(name='Recurring').exists():
+        ProjectStatus.objects.create(name='Recurring', has_end_date=False, has_start_date=True)
+        print("Recurring project status created.")
+    else:
+        print("Recurring project status already exists.")
+
+
+def create_organization_test_data(number_of_rows: int):
+    print("Creating orgranization data...")
+    for i in range(number_of_rows):
+        name = "Test Org {}".format(i)
+        url_slug = name.replace(" ", "")
+        if not Organization.objects.filter(name=name).exists():
+            Organization.objects.create(
+                name=name, url_slug=url_slug,
+                country="Germany", short_description="This is a test organization"
+            )
+            print("{} organization created.".format(name))
+        else:
+            print("{} orgranization already exists.".format(name))
+
+
+def create_project_test_data(number_of_rows: int):
+    print("Creating project data...")
+    for i in range(number_of_rows):
+        name = "Test project {}".format(i)
+        url_slug = name.replace(" ", "")
+        if not Project.objects.filter(name=name).exists():
+            Project.objects.create(
+                name=name, url_slug=url_slug,
+                status=ProjectStatus.objects.get(name="In Progress"),
+                start_date=timezone.now(),
+                short_description="This is a test project.",
+                country="Germany", collaborators_welcome=True
```

Comment:
I would add `city="Test {}".format(i)`. Otherwise, project cards will look a bit weird if the city is always empty.